gitmoji/index.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const core = require('@actions/core')
+const axios = require('axios').default
+
+axios.get('https://raw.githubusercontent.com/carloscuesta/gitmoji/master/src/data/gitmojis.json')
```

Comment:
Hey! I was taking a look at this out of curiousity😛 you can also import the list from the following npm package

https://www.npmjs.com/package/gitmojis

Reply:
@carloscuesta Ah thank you... Maybe this will solve the problem.. Not quite sure. I will look into this later in this week (if I get time)